OverlayPlugin.Core/MemoryProcessors/Party/PartyMemory65.cs

```diff
@@ -0,0 +1,227 @@
+        public unsafe partial struct GroupManager
+        {
+            [FieldOffset(0x0)]
+            public fixed byte PartyMembers[0x390 * 8]; // PartyMember type
+            [FieldOffset(0x1C80)]
+            public fixed byte AllianceMembers[0x390 * 20]; // PartyMember type
+            [FieldOffset(0x63C0)]
+            public uint Unk_3D40;
+            [FieldOffset(0x63C4)]
+            public ushort Unk_3D44;
+            [FieldOffset(0x63C8)]
+            public long PartyId; // both seem to be unique per party and replicated to every member
+            [FieldOffset(0x63D0)]
+            public long PartyId_2;
+            [FieldOffset(0x63D8)]
+            public uint PartyLeaderIndex; // index of party leader in array
+            [FieldOffset(0x63DC)]
+            public byte MemberCount;
+            [FieldOffset(0x63DD)]
+            public byte Unk_3D5D;
+            [FieldOffset(0x63DF)]
+            public byte Unk_3D5F; // some sort of count
+            [FieldOffset(0x63E0)]
+            public byte Unk_3D60;
+            [FieldOffset(0x63E1)]
+            public byte AllianceFlags; // 0x01 == is alliance; 0x02 == alliance with 5 4-man groups rather than 2 8-man
+        }
+        #endregion
+        [StructLayoutAttribute(LayoutKind.Sequential, Pack = 1)]
+        private struct DoubleGroupManager
+        {
+            public GroupManager groupManager1;
+            public GroupManager groupManager2;
+        }
+
+        public PartyMemory65(TinyIoCContainer container) : base(container) { }
+
+        public override Version GetVersion()
+        {
+            return new Version(6, 5);
+        }
+
+
+        public unsafe PartyListsStruct GetPartyLists()
+        {
+            if (!IsValid())
+            {
+                return new PartyListsStruct();
+            }
+
+            ScanPointers();
+
+            if (partyInstanceAddress.ToInt64() == 0)
+            {
+                return new PartyListsStruct();
+            }
+            var groupManager = ManagedType<DoubleGroupManager>.GetManagedTypeFromIntPtr(partyInstanceAddress, memory).ToType();
+
+            // `PartyMembers` is a standard array, members are moved up/down as they're added/removed.
+            // As such, limit extracting members to the current count to avoid "ghost" members
+            var partyMembers = extractPartyMembers(groupManager.groupManager1.PartyMembers, Math.Min((int)groupManager.groupManager1.MemberCount, 8));
+
+            // `AllianceMembers` is a fixed-position array, with removed elements being mostly zero'd out
+            // Easiest way to check if an entry is still active is to check for `Flags != 0`
+            var alliance1Members = extractAllianceMembers(groupManager.groupManager1.AllianceMembers, 20, 0, 8);
+            var alliance2Members = extractAllianceMembers(groupManager.groupManager1.AllianceMembers, 20, 8, 8);
+            // TOOD: Actually verify D/E/F alliance info?
```

Comment:
I think this is the part that makes me hesitant about D/E/F. It's a different object and sounds(?) untested. Is there some way to test this?

If you feel confident about leaving it in, then go for it. I don't feel that strongly here.

Reply:
I feel confident that leaving this in won't cause any issues, there is enough logic in place to detect/prevent bad data as far as party members being included in the return array. The only question mark here is more of an "is this actually assigned as D/E/F" question.

Reply:
Sounds good! We can sort it out later if this won't cause issues.